test/models/comment_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class CommentTest < ActiveSupport::TestCase
```

Comment:
Kindly write complete test for comment. Test for

1. Valid comments
2. Invalid comments
3. Associations

Refer to Rails Tutorial book for how to write proper tests in Minitest